api_server/src/lib.rs

```diff
@@ -1,21 +1,21 @@
 // Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-extern crate futures;
-extern crate hyper;
-extern crate serde;
+
```

Comment:
This file has way more whitespaces that it is needed.

The pattern we used so far:
- one (optional) empty line after the copyright (some files don't have empty lines after copyright) 
- external crate dependencies (like hyper and serde_derive in this file)
- one empty line
- internal crate dependencies
- one empty line
- imports from external crates
- empty line
- imports from crates from the same workspace (like fc_util)
- imports from the same crate